packages/jellyfish-api-core/__tests__/category/loan/updateLoanScheme.test.ts

```diff
@@ -0,0 +1,189 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+import { GenesisKeys } from '@defichain/testcontainers'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+
+    // Default scheme
+    await testing.container.call('createloanscheme', [100, new BigNumber(1.5), 'default'])
+    await testing.generate(1)
+  })
+
+  beforeEach(async () => {
+    await testing.container.call('createloanscheme', [200, new BigNumber(2.5), 'scheme1'])
+    await testing.generate(1)
+  })
+
+  afterEach(async () => {
+    const data = await testing.container.call('listloanschemes')
+    const result = data.filter((d: { default: boolean }) => !d.default)
+
+    for (let i = 0; i < result.length; i += 1) {
+      // Delete all schemes except default scheme
+      await testing.container.call('destroyloanscheme', [result[i].id])
+      await testing.generate(1)
+    }
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should updateLoanScheme', async () => {
+    const loanSchemeId = await testing.rpc.loan.updateLoanScheme(300, new BigNumber(3.5), { id: 'scheme1' })
```

Comment:
`updateLoanScheme` input parameter order is a bit odd.

Reply:
Ya. Fixing